### PR TITLE
docs: modify and translate schema-api

### PR DESCRIPTION
**Pull request by liuxiaocs7**
subtask of #109
as title

### CODE REVIEW

**Comment by liuxiaocs7** [comment]
The examples in `schema` info and [`load` demo](https://hugegraph.apache.org/docs/quickstart/hugegraph-loader/#331-mapping-file-overview) are the same, but I noticed that some types are different, so I replaced them all directly.

File: content/cn/docs/clients/restful-api/schema.md [hunk 1]
```diff
@@ -24,299 +26,297 @@ GET http://localhost:8080/graphs/hugegraph/schema
 
 ```json
 {
-    "propertykeys": [
-        {
```
Comment (imbajin) [inline content/cn/docs/clients/restful-api/schema.md:28]: need ensure the space for the json file is 2 not 4?

File: content/cn/docs/clients/restful-api/schema.md [hunk 2]
```diff
@@ -11,7 +11,9 @@ HugeGraph 提供单一接口获取某个图的全部 Schema 信息，包括：Pr
 ##### Method & Url
 
 ```
-GET http://localhost:8080/graphs/hugegraph/schema
+GET http://localhost:8080/graphs/{graph_name}/schema
+
+eg: GET http://localhost:8080/graphs/hugegraph/schema
```
Comment (imbajin) [inline content/cn/docs/clients/restful-api/schema.md:16]: maybe use `e.g` is better?

File: content/en/docs/clients/restful-api/schema.md [hunk 3]
```diff
@@ -6,12 +6,14 @@ weight: 1
...
-HugeGraph 提供单一接口获取某个图的全部 Schema 信息，包括：PropertyKey、VertexLabel、EdgeLabel 和 IndexLabel。
+HugeGraph provides a single interface to get all Schema information of a graph, including: PropertyKey, VertexLabel, EdgeLabel and IndexLabel.
 
 ##### Method & Url
 
 ```
-GET http://localhost:8080/graphs/hugegraph/schema
+GET http://localhost:8080/graphs/{graph_name}/schema
+
+eg: GET http://localhost:8080/graphs/hugegraph/schema
```
Comment (imbajin) [inline content/en/docs/clients/restful-api/schema.md:16]: same

**Comment by imbajin** [comment]
Also could fix it

image

TODO:
- [ ] add add a "Easy to start" task ==> (add Wechat picture in  README for each repo, also in the website)

**Comment by liuxiaocs7** [comment]
> Also could fix it: (the mail **should be** `dev@hugegraph.apache.org`, typo wrong in pic ⬇️)
> 
> image
> TODO:
> 
> * [ ]  add add a "Easy to start" task ==> (add **Wechat Picture** in  `README` for each repo, also in the website)

Good idea, will be done in another seperate PR.

File: content/cn/docs/clients/restful-api/schema.md (hunk 1 above)
Comment (liuxiaocs7) [reply content/cn/docs/clients/restful-api/schema.md]: done!

**Review by imbajin — APPROVED** [review]
LGTM (Have tested it with the example data?)

**Comment by liuxiaocs7** [comment]
> LGTM (Have been tested it?)

yeap, local tests run fine

**Pull request merged**